### DISCUSSION TITLE
这也太难了吧！！

### DISCUSSION THREAD

**Discussion by PraviDrror04** [share121/inter-knot#624] [其他]
![屏幕截图 2024-06-06 221149](https://github.com/user-attachments/assets/fa8d10a9-fc36-4606-b32d-4f9220952e7a)
最近备战小学生录取考试，有一题把我难住了，谁能教教我怎么做？？？

**Comment by lllliufeng** [comment]
实际点考个研究生得了，小学生不是我们能够碰瓷的，还有小黑子漏出犄角了吧
